tables.go

```diff
@@ -42,81 +44,79 @@ func (lx *LXRHash) Log(msg string) {
 // HashSize is the number of bits in the hash; truncated to a byte bountry
 // Passes is the number of shuffles of the ByteMap performed.  Each pass shuffles all byte values in the map
 func (lx *LXRHash) Init(Seed, MapSizeBits, HashSize, Passes uint64) {
+	// These panics are here to maintain the interface (using
+	// panic instead of returning an error) while reusing the
+	// code in InitFromPath.
```

Comment:
You're right.  They belong in the PR because you might want to know why there are panics where there should be errors returned.  They belong in the code for the same reason.  They explain why the code uses `panic` instead of returning an error like a normal go library.

That said, if you want fewer comments, I'll pull them.

Reply:
The problem is that comment won't be seen by any third party dev unless they look at the source code. How about appending the function-comment to include a third line:
```
// Panics if MapSizeBits is too low (<8)
```

That way it will show up in IDEs where the information is useful

Reply:
This is a good solution.